tests/args_parser.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: (GPL-2.0 OR Linux-OpenIB)
+# Copyright (c) 2020 Kamal Heib <kamalheib1@gmail.com>, Inc. All rights reserved.  See COPYING file
+
+import sys
+import argparse
+
+
+class ArgsParser(object):
+    def __init__(self):
+        self.args = None
+
+    def parse_args(self):
+        parser = argparse.ArgumentParser()
+        parser.add_argument('-d', '--dev_name', help='RDMA device to run the tests on')
+        ns, args = parser.parse_known_args()
+        self.args = vars(ns)
+        sys.argv[1:] = args
```

Comment:
This seems pretty clever.

In this case lets not use short '-d' strings due to possible conflict

Reply:
Fixed:
1. '-d' removed.
2. I've added --verbos per the request from @galpress.

Reply:
Thanks 😄